src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  --color-mockup-blue: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo__link {
+  cursor: pointer;
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
```

Comment:
it's redundant

Reply:
without display: flex and align-items: center

![image](https://user-images.githubusercontent.com/116076758/208380794-1f724457-12e4-447f-9127-5e21dca5215a.png)

Reply:
fixed